r4/src/main/java/org/opencds/cqf/r4/builders/RequestGroupActionBuilder.java

```diff
@@ -40,6 +42,24 @@ public RequestGroupActionBuilder buildDocumentation(List<RelatedArtifact> docume
         return this;
     }
 
+    public RequestGroupActionBuilder buildRelatedAction(
+            List<PlanDefinition.PlanDefinitionActionRelatedActionComponent> planDefinitionRelatedActions) {
+        List<RequestGroup.RequestGroupActionRelatedActionComponent> requestGroupRelatedActions = new ArrayList<>();
+
+        // Copy each PlanDefinition.RelatedAction into RequestGroup.RelatedAction
+        for (PlanDefinition.PlanDefinitionActionRelatedActionComponent planDefinitionRelatedAction : planDefinitionRelatedActions) {
+            String relationshipCode = planDefinitionRelatedAction.getRelationship().toCode();
+            RequestGroup.ActionRelationshipType relationship = RequestGroup.ActionRelationshipType
+                    .fromCode(relationshipCode);
+            RequestGroup.RequestGroupActionRelatedActionComponent requestGroupRelatedAction = new RequestGroup.RequestGroupActionRelatedActionComponent();
+            requestGroupRelatedAction.setActionId(planDefinitionRelatedAction.getActionId());
+            requestGroupRelatedAction.setRelationship(relationship);
+            requestGroupRelatedActions.add(requestGroupRelatedAction);
```

Comment:
Minor, but it would make sense to use the fluent interface here.
```suggestion
            RequestGroup.RequestGroupActionRelatedActionComponent requestGroupRelatedAction = new RequestGroup.RequestGroupActionRelatedActionComponent()
                .setActionId(planDefinitionRelatedAction.getActionId())
                .setRelationship(relationship);
```